src/java/mars/Simulation.java

```diff
@@ -1,25 +1,66 @@
 package mars;
 
+import mars.rockets.U1;
+import mars.rockets.U2;
+import mars.utils.Constans;
+import mars.utils.FileMars;
+
+import java.util.ArrayList;
 import java.util.List;
 
 public class Simulation {
 
     private List<Rocket> rockets;
 
-    public List<Item> loadItems(){
-        // implement code here
-        return null;
+    public Simulation() {
+        rockets = new ArrayList<>();
+    }
+
+    public List<Item> loadItems(String phase){
+        FileMars fileMars = new FileMars();
+        List<Item> items = new ArrayList<>();
+        List<String> itemsdata = fileMars.readFile(phase);
```

Comment:
Rename this field using CamelCase notation: https://sanaulla.info/2008/06/25/camelcase-notation-naming-convention-for-programming-languages/

I recommend you to use `itemsDataList` instead